app/global/MetricsPortalServerConfigStartup.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package global;
+
+import com.arpnetworking.steno.Logger;
+import com.arpnetworking.steno.LoggerFactory;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.Inject;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import io.ebean.config.ServerConfig;
+import io.ebean.event.ServerConfigStartup;
+
+/**
+ * Plugin class to configure Ebean's {@link ServerConfig} at runtime.
+ * <p>
+ * This is necessary for configuring the Ebean server with dependencies constructed via Guice, for instance. This
```

Comment:
nit: try to wrap comments at 80 characters.

Reply:
After seeing this I thought I could just set this option in IntelliJ. Unfortunately not only can you not set different margins for comments v. code, the ticket for this was opened [7 years ago](https://youtrack.jetbrains.com/issue/IDEA-114661).

In fact, another was opened [15(!!!) years ago](https://youtrack.jetbrains.com/issue/IDEABKL-1289)

I'll just do this manually...